snmp/datadog_checks/snmp/data/profiles/idrac.yaml

```diff
@@ -9,3 +9,45 @@ device:
   vendor: "dell"
 
 sysobjectid: 1.3.6.1.4.1.674.10892.*
+metadata:
```

Comment:
As discussed, I think before adding the metadata to idrac.yaml, we should check if devices with sysObjectID `1.3.6.1.4.1.674.10892.2` and `1.3.6.1.4.1.674.10892.5` require a separate profile or not.

Also, it might worth checking if devices can have `1.3.6.1.4.1.674.10892.1` (poweredge?) as sysObjectID or not. In case, it's not possible:
- do we still need `dell-poweredge.yaml` ?
- how do we monitor poweredge devices (metrics and metadata)

Reply:
Seems like `1.3.6.1.4.1.674.10892.2` is used for older DRAC and iDRAC version (v1-v7) while `1.3.6.1.4.1.674.10892.5` is used for newer versions of iDRAC only (v7-v9+). Also `10892.2` is used for Chassis Management Controller (CMC).

Based on this, I'd say we could have different profiles for each, **but**:
- I'm not sure what happens for iDRAC7, does it define both?
- There doesn't seem to be any relevant metadata we can fetch from `10892.2`: https://www.circitor.fr/Mibs/Html/D/DELL-RAC-MIB.php there is no metadata information for the server, only for CMC.

Devices can have a `.10892.1` OID, we have at least an example. I think it's when it's monitored via Dell EMC OpenManage instead of via it's iDRAC port. Also it's not clear how we can retrieve metadata from such a thing, as the exposed data is a table, with one row per server.

With all of that I'd say:
- Let's keep this metadata in the idrac profile. This will work for iDRAC7 and newer. `10892.5`
- For older DRAC/iDRAC `10892.2` I'm not sure we have the information we want in the MIB so let's skip it
- Let's keep the poweredge profile the way it is as it's not of much use.